base/loading.jl

```diff
@@ -347,9 +357,11 @@ include_string(txt::ByteString, fname::ByteString) =
     ccall(:jl_load_file_string, Any, (Ptr{UInt8},Csize_t,Ptr{UInt8},Csize_t),
           txt, sizeof(txt), fname, sizeof(fname))
 
-include_string(txt::AbstractString, fname::AbstractString) = include_string(bytestring(txt), bytestring(fname))
+include_string(txt::AbstractString, fname::AbstractString) =
```

Comment:
Just put `fname::AbstractString="string"` for the default, rather than an explicit second method below?